plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManager.java

```diff
@@ -46,6 +46,9 @@
     static final ConfigKey<Integer> s_vmwareOVAPackageTimeout = new ConfigKey<Integer>(Integer.class, "vmware.package.ova.timeout", "Advanced", "3600",
             "Vmware script timeout for ova packaging process", true, ConfigKey.Scope.Global, 1000);
 
+    static final ConfigKey<Boolean> s_vmwareCleanupPortGroups = new ConfigKey<Boolean>("Advanced", Boolean.class, "vmware.cleanup.port.groups",  "false",
+            "Remove unused port groups. WARNING: When set to true, native VMware HA might not work.", true, ConfigKey.Scope.Global);
```

Comment:
@Spaceman1984 If my understanding is correct, cleanup port groups happens only on the network cleanup only and the old uncleaned port groups of other networks (when this setting is false) still remains. If so, Can you mention in the setting description that port groups cleanup happens on the network cleanup for that network only.

Reply:
> Hi @Spaceman1984 there are some open questions on this PR, can you please advise?

Sure @nvazquez

Reply:
To my knowledge, this setting is not used through garbage collection, it is checked when an instance is destroyed.